vm-virtio/src/vhost_user/blk.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use libc;
+use libc::EFD_NONBLOCK;
+use std::cmp;
+use std::io::Write;
+use std::ptr::null;
+use std::sync::{Arc, RwLock};
+use std::thread;
+use std::vec::Vec;
+
+use crate::VirtioInterrupt;
+
+use vm_memory::GuestMemoryMmap;
+use vmm_sys_util::eventfd::EventFd;
+
+use super::super::{ActivateError, ActivateResult, Queue, VirtioDevice, VirtioDeviceType};
+use super::handler::*;
+use super::vu_common_ctrl::*;
+use super::{Error, Result};
+use std::mem;
+use vhost_rs::vhost_user::message::VhostUserConfigFlags;
+use vhost_rs::vhost_user::message::{VhostUserProtocolFeatures, VhostUserVirtioFeatures};
+use vhost_rs::vhost_user::{Master, VhostUserMaster, VhostUserMasterReqHandler};
+use vhost_rs::VhostBackend;
+use virtio_bindings::virtio_blk::*;
+
+macro_rules! offset_of {
+    ($ty:ty, $field:ident) => {
+        unsafe { &(*(null() as *const $ty)).$field as *const _ as usize }
+    };
+}
+
+struct SlaveReqHandler {}
+impl VhostUserMasterReqHandler for SlaveReqHandler {}
+
+#[derive(Copy, Clone, Debug, Default)]
+#[repr(C, packed)]
+struct virtio_blk_config {
```

Comment:
What's wrong with the version in virtio-bindings?

Reply:
https://github.com/intel/cloud-hypervisor/pull/292 addresses the issue, which means the virtio-bindings from 5.0 are up to date.